src/parser/test/test.ml

```diff
@@ -209,35 +236,35 @@ let%expect_test _ =
   let module Trivial = struct
     let empty =
       test "";
```

Comment:
Would it be possible to have test print something very special in case there is a non-empty frontmatter, and otherwise use the previous output?
And another `test_with_frontmatter` which output the current version of `test`.

It would prevent having multiple empty parentheses which makes it hard to read in 99% of the cases.

Reply:
Agreed.